modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/PlannerTest.java

```diff
@@ -87,6 +93,9 @@
 //@WithSystemProperty(key = "calcite.debug", value = "true")
 @SuppressWarnings({"TooBroadScope", "FieldCanBeLocal", "TypeMayBeWeakened"})
 public class PlannerTest extends AbstractPlannerTest {
+    /** */
+    private static final long PLANNER_TIMEOUT = 5_000;
```

Comment:
Let's use smaller timeout (1 sec for example) to make tests pass faster.

Reply:
Less is not enough to produce physical plan

Reply:
It confirms that using JOINs to emulate long planning it's a bad idea.